app/ts/components/ConnectAccount.tsx

```diff
@@ -105,7 +100,7 @@ const NetworkName = () => {
 		case 'pending':
 			return <AsyncText placeholderLength={8} />
 		case 'resolved':
-			const networkName = network.value.value.chainId === 1 ? 'mainnet' : network.value.value.name
+			const networkName = network.value.value.chainId === 1n ? 'mainnet' : network.value.value.name
```

Comment:
hmm what is the purpose of this? Why mainnet is not in the network value?

Reply:
this was indeed corrected in v6 as `mainnet`, was previously returning `homestead` removed the name mapping.